src/adera/machines/links.cpp

```diff
@@ -1,12 +1,63 @@
 #include "links.h"
 
+using namespace osp;
+
 using osp::link::MachTypeReg_t;
 using osp::link::MachTypeId;
 
 namespace adera
 {
 
-MachTypeId const gc_mtUserCtrl = osp::link::MachTypeReg_t::create();
-MachTypeId const gc_mtMagicRocket = osp::link::MachTypeReg_t::create();
+MachTypeId const gc_mtUserCtrl      = MachTypeReg_t::create();
```

Comment:
Are these supposed to be accessible outside of this translation unit?

If not, best to move them to an anonymous namespace and make them static, to communicate to the linker that they can't escape the translation unit other than via being returned as a pointer/reference.